apps/pie-storybook/stories/introduction/design-tokens-cookbook.mdx

```diff
@@ -0,0 +1,174 @@
+>
+> These examples are purely for illustrating how to use our design token variables. They are not intended as recommendations for how to structure your CSS or name your classes.
+
+## Table of Contents
+
+- [Colour](#colour)
+    - [Use brand colour as button background](#use-brand-colour-as-button-background)
+    - [Using HSL](#using-hsl)
+- [Spacing](#spacing)
+    - [Stack items with spacing token](#stack-items-with-spacing-token)
+    - [Add padding using spacing alias](#add-padding-using-spacing-alias)
+- [Typography](#typography)
+    - [Body text](#body-text)
+    - [Heading example](#heading-example)
+    - [Line height](#line-height)
+- [Radius](#radius)
+    - [Rounded card](#rounded-card)
+- [Motion](#motion)
+    - [Fade in-out transition](#fade-in-out-transition)
+- [Elevation](#elevation)
+    - [Card with elevation](#card-with-elevation)
+
+## Colour
+This example uses a semantic colour token (`--dt-color-interactive-brand`) to define the background, and a complementary content token (`--dt-color-content-interactive-primary`) for the text. These tokens abstract away raw hex or HSL values in favor of role-based names, which makes them more adaptable to different themes (like dark mode) and helps maintain accessibility and brand consistency.
+
+### Use brand colour as button background
+```css
+.box-brand {
+    background-color: var(--dt-color-interactive-brand);
+    color: var(--dt-color-content-interactive-primary);
+}
+```
+
+### Using HSL
+
+For greater flexibility, we provide `-h`, `-s`, and `-l` variants of all our colour tokens. These expose the hue, saturation, and lightness components separately, allowing you to compose colours using the native CSS `hsl()` function.
+
+You can combine these with our [state tokens](https://pie.design/foundations/colour/tokens/alias/light/#States), such as `--dt-color-hover-01`, which typically represents a percentage to dynamically adjust things like lightness for hover or focus states.
+
+> You will need to apply the state token to a `calc()` css function with `-1` to darken the colour.
+
+```css
+.hsl-example-darken {
+  background-color: hsl(var(--dt-color-container-default-h), var(--dt-color-container-default-s), calc(var(--dt-color-container-default-l) + calc(-1 * var(--dt-color-hover-01))));
+}
+```
+
+```css
+.hsl-example-lighten {
+    background-color: hsl(var(--dt-color-container-default-h), var(--dt-color-container-default-s), calc(var(--dt-color-container-default-l) + var(--dt-color-hover-01)));
+}
+```
+
+## Spacing
+
+### Stack items with spacing token
+This example uses a spacing token (`--dt-spacing-d`) with the CSS gap property to control vertical rhythm between stacked elements. By using tokens for spacing rather than hardcoded values, we ensure consistent layout behavior across all components, and make global spacing updates easier to apply.
```

Comment:
```suggestion
This example uses a spacing token (`--dt-spacing-d`) with the CSS gap property to control vertical rhythm between stacked elements. By using tokens for spacing rather than hardcoded values, we ensure consistent layout behaviour across all components and make global spacing updates easier to apply.
```